style/global.scss

```diff
@@ -82,3 +82,9 @@ h6 {
     flex-direction: column;
   }
 }
+
+nav * {
+  font-familY: var(--font-family-primary) !important;
```

Comment:
```suggestion
  font-family: var(--font-family-primary) !important;
```

typo

Reply:
Also curious to know why we need the `!important` here and what the considerations were around that?

Reply:
The `!important` doesn't need to be there, good call. I put it there for testing and forgot to remove it.